.github/labeler.yml

```diff
@@ -1,2 +1,2 @@
 shared-web:
-- webui/react/src/shared/**
+- webui/react/src/shared/**/*
```

Comment:
nit: i remember there was some comments about this in the last round

Reply:
reincorporated